bin/gdb_wrapper

```diff
@@ -65,7 +65,7 @@ unless options.ruby_path
   exit 1
 end
 
-argv = '["' + ARGV * '", "' + '"]'
+argv = '["' + ARGV * '", "'
```

Comment:
I think precalculating this part of argv (and creating such variable) is not really needed